app/Cmd.hs

```diff
@@ -0,0 +1,71 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Cmd (
+  CmdOpts (CmdOpts),
+  MongoOpts (MongoOpts)
+  ) where
```

Comment:
The common idiom in Haskell is to use leading commas.

```haskell
module Cmd
  ( CmdOpts (CmdOpts)
  , MongoOpts (MongoOpts)
  ) where
```

Of course, this is Haskell, and you're welcome to have whatever code style you want :smile:

Reply:
Is the opening parenthesis on a newline also a common idiom?

Reply:
It is, though I have seen it on the same line:

```haskell
module Cmd (
    CmdOpts (..)
  , MongoOpts (..)
  )
```

(this is true for record declarations and other sorts of comma-delimited lists)

Reply:
Should be fixed in 24e7bde2bcc268d64f3c32bbab00daa4b263af27.